checker/tests/nullness/LogRecordTest.java

```diff
@@ -0,0 +1,31 @@
+// Test Case for adding nullness annotations to java.util.logging.LogRecord
+
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+
+public class LogRecordTest {
+
+    @SuppressWarnings(
```

Comment:
You can avoid the need for this warning suppression by making `level` a parameter to test, as in:

```
void test(Level level) {
```

Reply:
Corrected.
Thanks for all your suggestions.
I will remember to follow them in the future.